### PR TITLE
Fix Product Configuration feature installation guide:

### PR DESCRIPTION
**Pull request by AsonUnique**
- Remove `details open` tags for consistency.
- Substitute the removed plugin with the actual one.
- Fix Widgets table markdown.
- Remove unneeded TwigConfig changes.
- Fixed small issues in code examples.

### CODE REVIEW

File: _includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md [hunk 1]
```diff
@@ -1988,30 +1979,6 @@ class AvailabilityDependencyProvider extends SprykerAvailabilityDependencyProvid
...
-     */
-    protected function getAvailabilityStrategyPlugins(): array
-    {
-        return [
-            new ExampleDateTimeConfiguratorAvailabilityStrategyPlugin(),
-        ];
-    }
-}
-```
-
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md:2014]: Why it was removed?

File: _includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md [hunk 2]
```diff
@@ -2133,46 +2100,10 @@ class ApplicationDependencyProvider extends SprykerApplicationDependencyProvider
...
-      );
-
-      $directories = array_merge(
-          $directories,
-          parent::getZedDirectoryPathPatterns()
-      );
-      sort($directories);
-      return $directories;
-  }
-}
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md:2169]: same here

File: _includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md (hunk 1 above)
Comment (AsonUnique) [reply _includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md:2014]: It is a duplicate.

File: _includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md (hunk 2 above)
Comment (dmiseev) [reply _includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md:2169]: Yes, but it's IG for projects, as I see comment on that file:
`Only needed in Project, not in demoshop` - we should keep it.

Comment (AsonUnique) [reply _includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md:2169]: No, these paths related only to `spryker/spryker` and `spryker/spryker-shop` repositories. Normally projects will have `'vendor/spryker/*/src/*/Zed/*/Presentation'` and `'vendor/spryker-shop/*/src/*/Zed/*/Presentation'` paths and they are covered by config on a core level.

**Review by dmiseev — DISMISSED** [review]

**Review by philinecao — APPROVED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**